### PR TITLE
theage.com.au, digminecraft.com, gamersnexus.net

### PR DESCRIPTION
**Pull request by DandelionSprout**
Example sites:

```
! https://www.theage.com.au/
theage.com.au##div[id^="adspot"]
! https://www.digminecraft.com/game_commands/connect_command.php
digminecraft.com##.slot
! https://www.gamersnexus.net/guides/3494-amd-ryzen-3000-undervolting-offset-override
gamersnexus.net##.moduleContent:has-text(/^\n\t\t\tAdvertisements:\n/)
```

### CODE REVIEW

File: dist/placeholder-buster.txt [hunk 1]
```diff
@@ -911,6 +911,15 @@ gamefaqs.gamespot.com###leader_top
...
 nulledpremium.com##.code-block:has(:scope > .adsbygoogle)
 
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/63
+theage.com.au##div[id^="adspot"]
+
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/63
+digminecraft.com##.slot
+
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/63
+gamersnexus.net##.moduleContent:has-text(/^\n			Advertisements:\n/)
```
Comment (jspenguin2017) [inline dist/placeholder-buster.txt]: Escape this (`\t`).

**Review by jspenguin2017 — CHANGES_REQUESTED** [review]

File: dist/placeholder-buster.txt (hunk 1 above)
Comment (DandelionSprout) [reply dist/placeholder-buster.txt]: Regex is a mysterious thing even *after* I've mostly learned it, but your idea seemed to work pretty well once I figured out how to apply it.

**Review by jspenguin2017 — APPROVED** [review]

**Comment by jspenguin2017** [comment]
`https://www.theage.com.au/`

**Comment by jspenguin2017** [comment]
`https://www.digminecraft.com/game_commands/connect_command.php`

**Comment by jspenguin2017** [comment]
`https://www.gamersnexus.net/guides/3494-amd-ryzen-3000-undervolting-offset-override`

**Pull request merged**